RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -30,6 +30,7 @@
 #include <TF1.h>
 #include <map>
 #include "RecoTauTag/RecoTau/interface/DeepTauScaling.h"
+#include "RecoTauTag/RecoTau/interface/DeepTauIdBase.h"
```

Comment:
Additional comment: most of includes can be removed as it repeats what is already included in `DeepTauIdBase.h`. Anyway, similarly as in other places I will deal with it myself.